python/cucim/setup.py

```diff
@@ -39,6 +39,7 @@ def read(*names, **kwargs):
     url='https://developer.nvidia.com/multidimensional-image-processing',
     packages=find_packages('src'),
     package_dir={'cucim': 'src/cucim'},
+    package_data={"": ["*.pyi", "*.h", "*.cu", "*.npy", "*.md", "*.txt"]},
```

Comment:
Think we can limit this to source code we want to include

Maybe `.npy` if it is used in tests or examples

Text files can probably be dropped

Basically this should include what we want installed in the users environment

```suggestion
    package_data={"": ["*.pyi", "*.h", "*.cu"]},
```